test/routes/template/templateAppendix.test.js

```diff
@@ -257,6 +260,221 @@ describe('/templates/{template}/appendix', () => {
   });
 });
 
+describe('/templates/{template}/appendix with project_id', () => {
```

Comment:
is appendix with null project id tested elsewhere?

Reply:
The default is null project id, so yes, all the original tests would be considered tests with null project id